frontend/lang/messages/en-US.json

```
@@ -596,12 +596,12 @@
     "create-recipe-description": "Create a new recipe from scratch.",
     "create-recipes": "Create Recipes",
     "import-with-zip": "Import with .zip",
-    "create-recipe-from-an-image": "Create Recipe from an Image",
-    "create-recipe-from-an-image-description": "Create a recipe by uploading an image of it. Mealie will attempt to extract the text from the image using AI and create a recipe from it.",
+    "create-recipe-from-an-image": "Create Recipe from Image(s)",
+    "create-recipe-from-an-image-description": "Create a recipe by uploading images of it. Mealie will attempt to extract the text from the image(s) using AI and create a recipe from it.",
```

For the instructions we can probably just keep it as "Images", I think users will understand they can use just one.

```suggestion
    "create-recipe-from-an-image": "Create Recipe from Images",
    "create-recipe-from-an-image-description": "Create a recipe by uploading images of the recipe text. Mealie will attempt to extract the text from the images using AI and create a new recipe from it.",
```

---

Also changed "of it" to "of the recipe text". Not really related to this PR but as long as we're changing it already there have been a handful of users who try to upload an image of the actual food rather than the recipe text.